frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -141,7 +142,34 @@ const Campaigns = () => {
           />
         )
       },
-      width: 'md center',
+      width: 'md',
+    },
+    {
+      name: '',
+      render: (campaign: Campaign) => {
+        if (
+          campaign.demoMessageLimit &&
+          ((numDemosSms === 0 && campaign.type === ChannelType.SMS) ||
+            (numDemosTelegram === 0 && campaign.type === ChannelType.Telegram))
+        ) {
+          return
```

Comment:
@NatMaeTan what is the behaviour/flow for "Duplicate" if the user tries to duplicate a demo campaign but does not have anymore demos left? 

I can imagine a case where a user tried out a few demo campaigns and finally decide to send it out for real. But then they find themselves unable to reuse/duplicate the demo campaign they've tested with because they no longer have anymore demos left.

Reply:
Agree that demo campaigns should have the button omitted. We shouldn't encourage duplicate of demo campaigns anyways!